builder.go

```diff
@@ -422,6 +426,10 @@ func (b *Builder) Run(step *Step, exec Executor, noRunsRemaining bool) error {
 	for _, run := range runs {
 		config := b.Config()
 		config.Env = step.Env
+		config.NetworkDisabled = (run.Network == "none")
```

Comment:
Shouldn't `config.NetworkDisabled` must be set on implementation end i.e buildah and imagebuilder should only parse `--network` and just set value in `run.Network` ? I always think of imagebuilder as the the frontend and buildah/docker as the actual implementation backend.

@nalind Could you correct if i got this wrong.

Reply:
The only field to pass information to buildah is the NetworkDisabled field.  Even in the --security=insecure, there is no Privileged field to set, so I use the SecurityOpts to pass it through.

Reply:
`imagebuilder` provides the parser in `dockerfile`, a builder in its top level package, and an executor that talks to an engine API (by way of go-dockerclient) that the builder can use in its `dockerclient` package.  Buildah uses the first two and supplies its own executor.